reward/reward_distributor.go

```diff
@@ -198,7 +198,7 @@ func GetBlockReward(header *types.Header, rules params.Rules, pset *params.GovPa
 	// some non-zero fee already has been paid to the proposer.
 	if !pset.DeferredTxFee() {
 		blockFee := GetTotalTxFee(header, rules, pset)
```

Comment:
Why do we call `GetTotalTxFee` again instead of using `pset.TotalFee`?

Reply:
`pset` is a governance parameter set (i.e., GovParamSet), so there's no TotalFee in it.

Reply:
Sorry for typo. What is mean is `spec.TotalFee` the previous value in the next line.

Reply:
Uhm... I understand now. the above logic only cares deferredRewards.

Reply:
When deferredFee is false, `spec.TotalFee` at line 201 is 0 due to line 237. However, line 202 updates `spec.TotalFee` to non-zero value. This caused the invariant mismatch (i.e., `minted + totalFee - burntFee != proposer + stakers + kgf + kir`).
https://github.com/klaytn/klaytn/blob/da11e9049a2fe889e505cbc2758ac94575bbcd7f/reward/reward_distributor.go#L201-L202
https://github.com/klaytn/klaytn/blob/da11e9049a2fe889e505cbc2758ac94575bbcd7f/reward/reward_distributor.go#L237

Instead of swapping the lines 201 and 202, it seems fair to increase the same `blockFee` amount from both LHS and RHS of the invariant.